src/functor.jl

```diff
@@ -13,7 +13,7 @@ function makefunctor(m::Module, T, fs = fieldnames(T))
     f in fs ? :(y[$(yᵢ += 1)]) : :(x.$f)
   end
   escfs = [:($f=x.$f) for f in fs]
-  
+
```

Comment:
A couple of no-op line changes. Do you know what that's from?

Reply:
Not sure, my editor may have trimmed existing trailing whitespace automatically on save?

Reply:
Oh yeah, so we'll keep these